axis_backend/apps/documents/tests/api/test_document_api.py

```
@@ -0,0 +1,559 @@
+            uploaded_by=self.user
+        )
+
+        response = self.client_api.get(f'/api/documents/?type={DocumentType.CONTRACT}')
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(response.data['count'], 1)
+        self.assertEqual(response.data['results'][0]['type'], DocumentType.CONTRACT)
+
+    def test_list_documents_with_status_filter(self):
+        """Test GET /api/documents/ with status filter."""
+        doc = Document.objects.create(
```

Ruff (0.14.5)

Local variable `doc` is assigned to but never used

Remove assignment to unused variable `doc`

(F841)